### PR TITLE
Xidicone.AgentSVN version 2.73

### PR DESCRIPTION
**Pull request by jussij**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/denelon/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/43327)

### CODE REVIEW

File: manifests/x/Xidicone/AgentSVN/2.73/Xidicone.AgentSVN.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# Created using wingetcreate 0.4.4.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: Xidicone.AgentSVN
+PackageVersion: 2.73
+InstallerSwitches:
+  Silent: /verysilent
+  SilentWithProgress: /silent
```
Comment (ItzLevvie) [inline manifests/x/Xidicone/AgentSVN/2.73/Xidicone.AgentSVN.installer.yaml:8]:
```suggestion
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

File: manifests/x/Xidicone/AgentSVN/2.73/Xidicone.AgentSVN.installer.yaml (hunk 1 above)
Comment (ItzLevvie) [reply manifests/x/Xidicone/AgentSVN/2.73/Xidicone.AgentSVN.installer.yaml:8]: This isn't needed because it's the default for `InstallerType: inno`

**Review by ItzLevvie — DISMISSED** [review]

**Comment by jussij** [comment]
Is there something wrong with this pull request?

I only ask because it appears to have sitting in a blocked state for several hours.

I'm planning to update a further two manifests with their latest software version information, but I'm holding of doing this until I know I've done this first update correctly.

**Comment by ryfu-msft** [comment]
@jussij, sorry for the delay, I had overlooked reviewing this PR. Once the build passes, you should be good to go :)

**Review by OfficialEsco — APPROVED** [review]

**Pull request merged**